fdbclient/VersionVector.h

```diff
@@ -69,6 +73,46 @@ struct VersionVector {
 		maxVersion = invalidVersion;
 	}
 
+	void getDelta(Version version, VersionVector& delta) const {
+		ASSERT(version <= maxVersion);
+
+		delta.clear();
+
+		if (version == maxVersion) {
+			return; // rerurn an invalid version vector
+		}
+
+		std::map<Version, std::set<Tag>> versionMap;
+		for (auto& iter : versions) {
+			if (iter.second > version) {
+				versionMap[iter.second].insert(iter.first);
+			}
+		}
+
+		for (auto& iter : versionMap) {
+			delta.setVersion(iter.second, iter.first);
+		}
+	}
+
+	void applyDelta(const VersionVector& delta) {
+		if (delta.maxVersion == invalidVersion) {
```

Comment:
Related to the `maxVersion` not been set in `getDelta()`.

Reply:
Yes.